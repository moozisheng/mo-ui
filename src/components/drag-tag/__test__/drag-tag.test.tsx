
import { render, cleanup, fireEvent, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import React from 'react';
import DragTag from '../index';

afterEach(cleanup);

test('renders TagList with base use', () => {
   const items = [
    {
      id: 1,
      text: 'Tag 1',
    },
    {
      id: 2,
      text: 'Tag 2',
    },
    {
      id: 3,
      text: 'Tag 3',
    },
  ]
  const { container } = render(<DragTag items={items} color="success" />);
  expect(container).toMatchSnapshot();
});








