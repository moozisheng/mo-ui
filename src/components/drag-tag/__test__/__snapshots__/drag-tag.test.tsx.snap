// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TagList with base use 1`] = `
<div>
  <span
    class="ant-tag ant-tag-success css-dev-only-do-not-override-2i2tap"
    style="cursor: move; transition: unset;"
  >
    Tag 1
  </span>
  <span
    class="ant-tag ant-tag-success css-dev-only-do-not-override-2i2tap"
    style="cursor: move; transition: unset;"
  >
    Tag 2
  </span>
  <span
    class="ant-tag ant-tag-success css-dev-only-do-not-override-2i2tap"
    style="cursor: move; transition: unset;"
  >
    Tag 3
  </span>
  <div
    id="DndDescribedBy-0"
    style="display: none;"
  >
    
    To pick up a draggable item, press the space bar.
    While dragging, use the arrow keys to move the item.
    Press space again to drop the item in its new position, or press escape to cancel.
  
  </div>
  <div
    aria-atomic="true"
    aria-live="assertive"
    id="DndLiveRegion-0"
    role="status"
    style="position: fixed; width: 1px; height: 1px; margin: -1px; border: 0px; padding: 0px; overflow: hidden; clip-path: inset(100%); white-space: nowrap;"
  />
</div>
`;
